Session_2/calc_function.js

```diff
@@ -0,0 +1,62 @@
+// Calculating with Functions
+// http://www.codewars.com/kata/calculating-with-functions
+
+function zero() {
```

Comment:
rename the functions

zero() virtually can be a generic method

Reply:
I understand how function renaming is important. but this was a problem statement in codewars.com which is calling the methods internally and checking the test cases. If I rename these all test cases are going to fail.
Codewars is a competitive coding platform with JS support.  
http://www.codewars.com/kata/calculating-with-functions